plugins/pur/core/pom.xml

```diff
@@ -38,7 +38,7 @@
     <!-- Test dependencies -->
     <se-jcr.version>0.9</se-jcr.version>
     <jcr.version>2.0</jcr.version>
-    <jackrabbit-core.version>2.14.2</jackrabbit-core.version>
+    <jackrabbit-core.version>2.16.5</jackrabbit-core.version>
```

Comment:
Please move the version property to the parent pom repository and use dependency management.